roles/postgresql/tasks/main.yml

```diff
@@ -81,24 +131,27 @@
     dest: '/etc/postgresql/{{ postgres_version }}/main/conf.d/statistics.conf'
     group: postgres
     owner: postgres
-    mode: 0644
+    mode: "0644"
   register: configure_postgres
-  when: postgresql_is_local
+  when:
+    - postgresql_is_local
+    - ansible_os_family == "Debian"
   notify: restart local postgres
 
 - name: PostgreSQL | rotate the logs for postgresql
-  template:
+  ansible.builtin.template:
```

Comment:
Do both PostgreSQL installations write logs to the same place? Or do we need to make sure the logrotate template can handle two locations?